### PR TITLE
add support for TS-XW initialization

### PR DESCRIPTION
**Pull request by yassineimounachen**
Output:

```
yassine@archlinux ~/hid-tmff2 (master)> sudo dmesg -W
[255365.548460] usb 3-1: new full-speed USB device number 6 using xhci_hcd
[255365.714021] usb 3-1: New USB device found, idVendor=044f, idProduct=b691, bcdDevice= 1.01
[255365.714026] usb 3-1: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[255365.714028] usb 3-1: Product: Thrustmaster TS-XW Racer GIP Wheel
[255365.714030] usb 3-1: Manufacturer: Thrustmaster
[255365.714032] usb 3-1: SerialNumber: 0000XXXXXXXXXXXX
[255370.805385] usb-thrustmaster 3-1:1.0: setup data couldn't be sent
[255370.807026] usb-thrustmaster 3-1:1.0: Wheel with (model, attachment) = (0x7, 0xa) is a Thrustmaster TS-XW. attachment_found=1
[255370.808122] usb 3-1: USB disconnect, device number 6
[255370.808216] usb-thrustmaster 3-1:1.0: URB to change wheel mode seems to have failed, error code -2
[255371.431808] usb 3-1: new full-speed USB device number 7 using xhci_hcd
[255371.597627] usb 3-1: New USB device found, idVendor=044f, idProduct=b692, bcdDevice= 1.00
[255371.597631] usb 3-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[255371.597633] usb 3-1: Product: Thrustmaster TS-XW Racer
[255371.597634] usb 3-1: Manufacturer: Thrustmaster
[255371.616734] input: Thrustmaster Thrustmaster TS-XW Racer as /devices/pci0000:00/0000:00:08.1/0000:10:00.3/usb3/3-1/3-1:1.0/0003:044F:B692.0008/input/input31
[255371.616808] tmff2 0003:044F:B692.0008: input,hidraw7: USB HID v1.11 Joystick [Thrustmaster Thrustmaster TS-XW Racer] on usb-0000:10:00.3-1/input0
[255371.643668] tmff2 0003:044F:B692.0008: force feedback for TS-XW
```

### CODE REVIEW

**Comment by yassineimounachen** [comment]
@Kimplul, just letting you know that it does work despite the shown errors.

File: tminit.c [hunk 1]
```diff
@@ -214,7 +215,7 @@ static void thrustmaster_model_handler(struct urb *urb)
 				tm_wheel->response->type);
 		return;
 	}
-
+
```
Comment (Kimplul) [inline tminit.c]: Please remove this extra tab.

**Comment by Kimplul** [comment]
Huh, interesting. For whatever reason I never received any emails about this PR, neither about it being opened or about your comment. Apparently your edit did something to kick GitHub's notification system into gear. I double-checked, this was the first email in my inbox.

> just letting you know that it does work despite the shown errors.

Yep, that's fine, the wheel just doesn't really follow USB protocol and the USB subsystem sometimes returns errors even though everything went well.

Really sorry it took this long to react.

**Comment by yassineimounachen** [comment]
Hehe, no worries. Thanks for checking the code.

**Pull request merged**